.coveragerc

```diff
@@ -48,6 +48,7 @@ omit =
     homeassistant/components/downloader.py
     homeassistant/components/keyboard.py
     homeassistant/components/light/hue.py
+    homeassistant/components/light/mqtt.py
```

Comment:
ok i will write the unittest. What is with the comment from @tomduijf ? Should i rewrite my code with one class?